src/components/structures/RoomView.js

```diff
@@ -803,6 +803,96 @@ module.exports = React.createClass({
         scrollNode.scrollTop = scrollNode.scrollHeight;
     },
 
+    // scroll the event view to put the given event at the bottom.
+    //
+    // pixel_offset gives the number of pixels between the bottom of the event
+    // and the bottom of the container.
+    scrollToEvent: function(event_id, pixel_offset) {
```

Comment:
please can we use camelCase in JS?  (there's some older snake_case stuff hanging around, but it should be defunct)